interfaces.go

```diff
@@ -315,6 +318,12 @@ type Event struct {
 	Spans           []*Span          `json:"spans,omitempty"`
 	TransactionInfo *TransactionInfo `json:"transaction_info,omitempty"`
 
+	// The fields below are only relevant for crons/check ins
+
+	CheckInID     string         `json:"check_in_id,omitempty"`
+	CheckIn       *CheckIn       `json:"check_in,omitempty"`
+	MonitorConfig *MonitorConfig `json:"monitor_config,omitempty"`
```

Comment:
If I follow how transactions are keeping their data, it would mean that every entry on `CheckIn` would be spread out and might mess with other fields in the future, so I keep it enclosed here with `omitempty` tag.